spec/features/admin/admin_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'feature_helper'
+
+describe 'ADMIN: AdminUsers' do
+  before { sign_in_as_admin }
+
+  let(:business) { create(:business_complete ) }
+  let(:user) { create(:user, business: business) }
+
+  it 'Creates an admin user' do
+    click_link 'Admin Users'
+    click_on 'New Admin User'
+    fill_in 'Email', with: 'test@example.com'
+    fill_in 'Password', with: 'password'
+    click_on 'Create Admin user'
+    expect(page).to have_content('Admin user was successfully created.')
+  end
+
+  it 'Edits an admin user' do
+    create(:admin_user, email: 'test@example.com', password: 'password')
+    click_link 'Admin Users'
+    click_link 'test@example.com'
+    click_on 'Edit Admin User'
+    fill_in 'Password', with: 'password'
+    click_on 'Update Admin user'
+    expect(page).to have_content('Admin user was successfully updated.')
+  end
+
+  @javascript
```

Comment:
Tests for Admin User create/edit/delete, there wasn't any test code.
The issue would have appeared if there was test code.